docker/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+cd /app/src
+
+cp -rT /usr/share/javascript/mathjax /app/static/mathjax
```

Comment:
Since /app/static is not a volume, this can be executed in the build. (Then, libjs-mathjax can also be uninstalled.)